### PR TITLE
PMM-2164: fix log events

### PR DESCRIPTION
**Pull request by arvenil**
Sorry guys but I have no idea how to properly fix it. This seems to work, but takes 1m 30s to run. Plus I believe it's either incorrect or there is something wrong with aws driver.

Second pair of eyes would be nice, please take a look.

EDIT: Adding `StartTime` seems to help. Can you verify this works?

### CODE REVIEW

File: enhanced/enhanced.go [hunk 1]
```diff
@@ -166,19 +166,10 @@ func (e *Exporter) collectValues(ch chan<- prometheus.Metric, instance config.In
...
-			defer wg.Done()
-
-			err := e.collectValue(ch, instance, key, value, l)
-			if err != nil {
-				log.Error(err)
-			}
-		}(key, value)
+		if err = e.collectValue(ch, instance, key, value, l); err != nil {
+			log.Error(err)
+		}
```
Comment (arvenil) [inline enhanced/enhanced.go]: This was requested in https://jira.percona.com/browse/PMM-1638 `Scrape all instance metrics in parallel.`

Comment (AlekSi) [reply enhanced/enhanced.go]: It was about doing required API calls in parallel, not light CPU work.

File: enhanced/enhanced.go [hunk 2]
```diff
@@ -140,26 +140,27 @@ func (e *Exporter) collectValues(ch chan<- prometheus.Metric, instance config.In
...
 
 		v, ok := message.(map[string]interface{})
 		if !ok {
-			return !lastPage
+			return
 		}
-		for i := range v {
-			values[i] = v
+		for key, value := range v {
+			values[key] = value
```
Comment (arvenil) [inline enhanced/enhanced.go]: 🤦‍♂️ true

File: enhanced/enhanced.go (hunk 1 above)
Comment (arvenil) [reply enhanced/enhanced.go]: There were no api calls when scraping metrics for given instance.

**Pull request merged**